docs/get-started/install.md

```diff
@@ -2,24 +2,21 @@
 title: 'Install Storybook'
 ---
 
+Use the Storybook CLI to install it in a single command. Run this inside your _existing project’s_ root directory:
+
 <!-- prettier-ignore-start -->
 
-<FeatureSnippets
+<CodeSnippets
   paths={[
-   'get-started/installation-command-section/angular.mdx',
-   'get-started/installation-command-section/ember.mdx',
-   'get-started/installation-command-section/html.mdx',
-   'get-started/installation-command-section/preact.mdx',
-   'get-started/installation-command-section/react.mdx',
-   'get-started/installation-command-section/svelte.mdx',
-   'get-started/installation-command-section/vue.mdx',
-   'get-started/installation-command-section/web-components.mdx',
-   'get-started/installation-command-section/qwik.mdx',
+   'common/init-command.npx.mdx',
+   'common/init-command.pnpm.mdx',
   ]}
 />
 
 <!-- prettier-ignore-end -->
 
+If you run into issues with the installation, check the [Troubleshooting section](#troubleshooting) below for guidance on how to solve it.
```

Comment:
Can we wrap this in an `.aside` so that it looks like secondary text.

```suggestion
<div class="aside">
If you run into issues with the installation, check the [Troubleshooting section](#troubleshooting) below for guidance on how to solve it.
</div>
```

Reply:
✅ Done!